### PR TITLE
Update to latest datachannel SDP. Fixes #6

### LINKED ISSUE #6: sctpmap application protocol reference for data channel m-line
Section 5.2.1
   Within the data m= section, the "a=mid", "a=ice-ufrag", "a=ice-
   passwd", "a=ice-options", "a=candidate", "a=fingerprint", and
   "a=setup" lines MUST be included as mentioned above, along with an
   "a=sctpmap" line referencing the SCTP port number and specifying the
   application protocol indicated in [I-D.ietf-rtcweb-data-protocol].
   [OPEN ISSUE:  the -01 of this document is missing this information.]

### PR DESCRIPTION
**Pull request by ekr**

### CODE REVIEW

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -1561,16 +1561,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+          <xref target="I-D.ietf-mmusic-sctp-sdp"></xref>, Section 4.1.
+          </t>
           <t>Within the data m= section, the "a=mid",
           "a=ice-ufrag", "a=ice-passwd", "a=ice-options", "a=candidate",
           "a=fingerprint", and "a=setup" lines MUST be included as mentioned
-          above, along with an "a=sctpmap" line referencing the SCTP port number
-          and specifying the application protocol indicated in <xref
-          target="I-D.ietf-rtcweb-data-protocol"></xref>. [OPEN ISSUE: the -01
-          of this document is missing this information.]</t>
+          above, along with an "a="fmtp:webrtc-datachannel"
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: remove extra quote

**Comment by fluffy** [comment]
Did not carefully review but looks fine to me

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -2069,18 +2068,16 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+          exactly match the fields in the offer.</t>
+
           <t>Within the data m= section, the "a=mid",
           "a=ice-ufrag", "a=ice-passwd", "a=ice-options", "a=candidate",
           "a=fingerprint", and "a=setup" lines MUST be included as mentioned
-          above, along with an "a=sctpmap" line referencing the SCTP port number
-          and specifying the application protocol indicated in <xref
-          target="I-D.ietf-rtcweb-data-protocol"></xref>. [OPEN ISSUE: the -01
-          of this document is missing this information.]</t>
+          above, along with an "a="fmtp:webrtc-datachannel"
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: see above

File: draft-ietf-rtcweb-jsep.xml [hunk 3]
```diff
@@ -1561,16 +1561,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+          <xref target="I-D.ietf-mmusic-sctp-sdp"></xref>, Section 4.1.
+          </t>
           <t>Within the data m= section, the "a=mid",
           "a=ice-ufrag", "a=ice-passwd", "a=ice-options", "a=candidate",
           "a=fingerprint", and "a=setup" lines MUST be included as mentioned
-          above, along with an "a=sctpmap" line referencing the SCTP port number
-          and specifying the application protocol indicated in <xref
-          target="I-D.ietf-rtcweb-data-protocol"></xref>. [OPEN ISSUE: the -01
-          of this document is missing this information.]</t>
+          above, along with an "a=fmtp:webrtc-datachannel"
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: I think you want the word "line" at the end of this line. (same below)

**Comment by juberti** [comment]
lgtm

**Pull request merged**